docs/concepts/notifications.md

```diff
@@ -92,7 +92,29 @@ function.run(handler=handler, notifications=[notification])
 ```
 
 ## Configuring Notifications For Pipelines
-For pipelines, you configure the notifications on the project notifiers. For example:
+To set notifications on pipelines, supply the notifications in the `run` method of either the project or pipeline.
```

Comment:
To set notifications on pipelines, supply the notifications in the `run` method of either the project or the pipeline.